templates/base.html

```diff
@@ -42,7 +42,7 @@
             <meta property="og:title" content="CCExtractor CI Platform" />
             <meta property="og:description" content="The Sample Platform is a Testing Platform for CCExtractor."/>
             <meta property="og:type" content="website" />
-            <!-- <meta property="og:url" content="https://sampleplatform.ccextractor.org{{ request.path }}"/> -->
+            <meta property="og:url" content="{{request.endpoint}}{{ request.path }}"/> 
```

Comment:
this will be still wrong. as for example: page https://sampleplatform.ccextractor.org/sample/
will output **<meta property="og:url" content="sample.index/sample/" />**